### PR TITLE
Prevents surgery infections for Abductors

### LINKED ISSUE #9199: Abductors are not equipped to perform heart surgery without infection
**Problem Description**:
What is the problem?

Recent bugfix has made gloves and surgical mask a necessity to avoid having a patient die near instantly during heart surgery. Abductors are also affected, and they have neither gloves nor mask in their ship or inventory.

**What did you expect to happen**: 
Why do you think this is a bug?

Perform surgery, go on your merry alien way.

**What happened instead**:
How is what happened different from what you expected?

Abductee dies of infection.

**Why is this bad/What are the consequences:**
Why do you think this is an important issue?

Surgery is a mandatory step of the abductor gameplay loop. Abductors shouldn't have to steal station supplies to be able to do it.

**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.

1. Abduct someone
2. Perform experimental surgery without gloves or surgical mask
3. Patient get infected and dies

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.

Tested on both local and live server.

### PR DESCRIPTION
**Pull request by datlo**
Fixes #9199

Adds spaceacillin to the reagents injected by the abductor surgery table, preventing infections. 

🆑
rscadd: Provided abductors with an infection free surgery table.
/🆑

### CODE REVIEW

**Comment by tigercat2000** [comment]
Not sure it's worth doing this way, given how absolutely ridiculous the surgery infection is (and the fact that it's not been enabled for a long time due to bugs), it's pretty likely it's gonna get ripped back out.

**Comment by datlo** [comment]
I believe it's still worth adding at least the gloves and space cleaner, gloves are also insulated so definitely helpful. Abductors need more utility gear so that they don't have to steal so much from the station in general.

I would have added the gloves and cleaner only, but right now the infection happens even with gloves equipped so the mask is necessary.

If the infection is removed/tamed down again, I'll update the PR to remove the surgical masks.

**Comment by Crazylemon64** [comment]
You're able to just add spaceacillin to the list of reagents the abductor table injects automatically, which would also take care of any infection troubles

**Comment by Crazylemon64** [comment]
https://github.com/ParadiseSS13/Paradise/blob/master/code/game/gamemodes/miniantags/abduction/abduction_gear.dm#L596
Add `spaceacillin` to this list, and infections will automatically be removed.

**Comment by taukausanake** [comment]
I think I like @Crazylemon64's idea better. I would personally like abductors to have alien-grade anti-germ equipment for surgery, but maybe they're advanced enough that they don't need it and just let the table do it for them.

I'd like surgery infections to be a thing. There can never be enough potential death in SS13

**Comment by Fox-McCloud** [comment]
Crazy's idea is a lot better and definitely fits more of the "naturally advanced; things happen without really trying" vibe.

**Comment by datlo** [comment]
Added spaceaccilin to the reagents injected, removed surgical masks and space cleaner, updated the PR to reflect changes.

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 1]
```diff
@@ -232,6 +232,7 @@
 	agent.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat(agent), slot_shoes)
 	agent.equip_to_slot_or_del(new /obj/item/clothing/under/color/grey(agent), slot_w_uniform) //they're greys gettit
 	agent.equip_to_slot_or_del(new /obj/item/storage/backpack(agent), slot_back)
+	agent.equip_to_slot_or_del(new /obj/item/clothing/gloves/combat, slot_gloves)
```
Comment (Crazylemon64) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Please remove the combat gloves from this PR, and I'll merge this promptly.

**Comment by datlo** [comment]
I still think abductors should get the gloves, but I suppose this is a bit outside the purpose of this PR at this point, I might make another PR for abductor gear eventually.

I removed the gloves leaving only the spaceaccilin

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 2]
```diff
@@ -269,7 +269,6 @@
 		console.gizmo = G
 		G.console = console
 	scientist.equip_to_slot_or_del(G, slot_in_backpack)
-
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: this is really nitpicky, but please re-add this new line

**Pull request merged**